packages/cerebral-router/src/index.js

```diff
@@ -38,7 +38,7 @@ export default function Router (options = {}) {
       throw new Error('Cerebral router - mapper option must be provided.')
     }
 
-    const routesConfig = flattenConfig(passedRoutesConfig)
+    const routesConfig = options.routes || flattenConfig(passedRoutesConfig)
```

Comment:
Wouldn't it be frustrating that modules level config would be ignored when also routes passed to router itself?
I still feel sceptic about allowing modules level routes by default. It ok as extension but too opinionated for default